lib/facter/jenkins.rb

```diff
@@ -12,3 +12,8 @@
     plugins.keys.sort.map { |plugin| "#{plugin} #{plugins[plugin][:plugin_version]}" }.join(', ')
   end
 end
+Facter.add(:jenkins_version) do
+    setcode do
+        Facter::Util::Resolution.exec("rpm -qa|grep jenkins|cut -d'-' -f2")
```

Comment:
This only works on Red Hat based systems. It'll also fail on the first run because the RPM is not yet installed. On upgrades it will fail because of a mismatch

Reply:
yes, sorry, you are right